src/app/core/ui-helper/entity-subrecord/entity-subrecord.component.ts

```diff
@@ -231,4 +234,11 @@ export class EntitySubrecordComponent implements OnInit, OnChanges, OnDestroy {
   isReadonlyInputType(inputType: ColumnDescriptionInputType): boolean {
     return inputType === ColumnDescriptionInputType.FUNCTION || inputType === ColumnDescriptionInputType.READONLY;
   }
+
+  transformInput(height: number, entity: Entity, colName: string): number {
+    if (this.transformFlag && entity.getType() === 'HealthCheck' && colName === 'height') {
+      height = height * 30.48;
+    }
+    return height;
+  }
```

Comment:
Adding the explicit transformation function for HealthCheck to the generic EntitySubrecordComponent is not ideal. How about instead extending the `ColumnDescription` with an optional inputTransformer property similar to the `formatter`?
 https://github.com/Aam-Digital/ndb-core/blob/9c34a807453fbae4934eb3ac7a497087fdc96078/src/app/core/ui-helper/entity-subrecord/column-description.ts#L43

That could also allow us to remove the `@Input() transformFlag` here which would better apply to a single column anyway.